kde-apps/akregator/akregator-9999.ebuild

```diff
@@ -19,39 +19,32 @@ KEYWORDS=""
 IUSE="speech telemetry"
 
 RDEPEND="
-	>=dev-libs/ktextaddons-1.4.42:6
-	>=dev-qt/qtbase-${QTMIN}:6[dbus,gui,network,widgets,xml]
```

Comment:
```
libQt6DBus.so.6 => /usr/lib64/libQt6DBus.so.6
libQt6Xml.so.6 => /usr/lib64/libQt6Xml.so.6
```

Reply:
After searching I see explicit usage despite not being explicitly added to find_package or being linked. They are mostly likely hoped to be transient dependencies.

Removing changes to the use.

Reply:
Ideally would be fixed upstream as well...